src/interfaces/v2.ts

```diff
@@ -0,0 +1,380 @@
+import type { JSONSchema7, JSONSchema7Type } from "json-schema";
+
+export type AsyncAPIVersion = '2.0.0' | '2.1.0' | '2.2.0' | '2.3.0' | '2.4.0';
```

Comment:
You can play with `@asyncapi/specs` package like we do in https://github.com/asyncapi/parser-js/pull/583/files#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R19 instead of hardcoding, so if we release `2.5.0` you don't need to update this line.

Reply:
Package `@asyncapi/specs` hasn't exported packages so we cannot "retrieve" types like here https://github.com/asyncapi/parser-js/pull/583/files#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R19

TS is a something like linter, it doesn't read types from runtime.

Reply:
We can also change that type to string.

Reply:
Yeah, I would vote for leaving it as string until we refactor `@asyncapi/specs` to export the types from it (if we do at some point).

Reply:
@smoya Done :)